.scripts/install.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -euo pipefail
```

Comment:
You might want to add `-x` which prints commands and their arguments as they are executed.